src/components/avatar-replay.js

```diff
@@ -0,0 +1,36 @@
+import botRecording from "../assets/avatars/bot-recording.json";
+
+AFRAME.registerComponent("avatar-replay", {
+  schema: {
+    camera: { type: "selector" },
+    leftController: { type: "selector" },
+    rightController: { type: "selector" }
+  },
+  init: function() {
+    const { camera, leftController, rightController } = this.data;
+
+    camera.setAttribute("motion-capture-replayer", { loop: true });
+    this._setupController(leftController);
+    this._setupController(rightController);
+
+    this.el.addEventListener("model-loaded", () => {
+      const cameraReplayer = camera.components["motion-capture-replayer"];
+      cameraReplayer.startReplaying(botRecording.camera);
+      const leftControllerReplayer = leftController.components["motion-capture-replayer"];
+      leftControllerReplayer.startReplaying(botRecording.left);
+      const rightControllerReplayer = rightController.components["motion-capture-replayer"];
+      rightControllerReplayer.startReplaying(botRecording.right);
+    });
+  },
+  _setupController: function(controller) {
```

Comment:
perhaps these blacklisted components should be moved into a const list at the top of the file?

Reply:
also sorry but I have to ask :) what is up with the `hand-controls2`?

Reply:
(part of the reason i ask is if we change it, we could just have a convention where we strip `-controls` components. might be too magical though)

Reply:
Sounds to magical to me. I don't like hand-controls2 either, but we called it that when we had to make modifications to A-Frame's hand-controls but weren't sure if it was worth contributing back.

`const` list sounds like a good idea though